closure/goog/i18n/numberformat.js

```diff
@@ -86,8 +86,8 @@ goog.i18n.NumberFormat = function(pattern, opt_currency, opt_currencyStyle) {
   // The multiplier for use in percent, per mille, etc.
   /** @private {number} */
   this.multiplier_ = 1;
-  /** @private {number} */
-  this.groupingSize_ = 3;
+  /** @private {Array} */
```

Comment:
You need to add a descriptive comment here for this new class attribute. What does it do? Where is it used? Etc.